src/test/fuzz/package_eval.cpp

```diff
@@ -300,5 +303,21 @@ FUZZ_TARGET(tx_package_eval, .init = initialize_tx_pool)
     UnregisterSharedValidationInterface(outpoints_updater);
 
     WITH_LOCK(::cs_main, tx_pool.check(chainstate.CoinsTip(), chainstate.m_chain.Height() + 1));
+    LOCK(tx_pool.cs);
+    for (const auto& tx_info : tx_pool.infoAll()) {
```

Comment:
did you want to use `CheckMempoolInvariants` here as well?

Reply:
Pulled into a helper function and reused :+1: